crates/scroll/rpc/src/eth/pending_block.rs

```diff
@@ -63,13 +64,12 @@ where
         &self,
         parent: &SealedHeader<ProviderHeader<Self::Provider>>,
     ) -> Result<<Self::Evm as ConfigureEvm>::NextBlockEnvCtx, Self::Error> {
-        Ok(NextBlockEnvAttributes {
-            timestamp: parent.timestamp().saturating_add(12),
+        Ok(ScrollNextBlockEnvAttributes {
+            timestamp: parent.timestamp().saturating_add(3),
```

Comment:
given that block time is configurable on the RN what is the impact of hardcoding the value here? Shall we create an issue to track this?

Reply:
I'm not too sure why this is done in this way to be honest, checking at the code for [op](https://github.com/scroll-tech/reth/blob/90d50903bc84e7163e41b634af721c8ce221b658/crates/optimism/rpc/src/eth/pending_block.rs#L69) and [eth](https://github.com/scroll-tech/reth/blob/56df19673678d25457aedbcbc2be07aa0aa6fbc1/crates/rpc/rpc/src/eth/helpers/pending_block.rs#L69), it looks like these values are just hardcoded. Op also uses 12 seconds for some reason.

Reply:
https://github.com/scroll-tech/reth/issues/215